v2/thirdpartyemailpassword/common-customizations/reset-password/post-reset-password.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: post-reset-password
+title: Post password reset
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./thirdpartyemailpassword/common-customizations/reset-password/post-reset-password.mdx -->
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import TabItem from '@theme/TabItem';
+
+# Doing operations post a password reset
+
+To perform any task like analytics, sending a user an email, notifying an internal dashboard, post resetting a password, you'll need to override the `passwordResetPOST` API.
+
+<BackendSDKTabs>
+<TabItem value="nodejs">
+
+```jsx
+SuperTokens.init({
+    appInfo: {...},
+    recipeList: [
+        ^{recipeNameCapitalLetters}.init({
+            // highlight-start
+            override: {
+                apis: (originalImplementation) => {
+                    return {
+                        ...originalImplementation,
+                        passwordResetPOST:async (input) => {
+                            
+                            if (originalImplementation.passwordResetPOST === undefined) {
+                                throw Error("original implementation of the passwordResetPOST API is undefined");
+                            }
+
+                            // First we call the original implementation
+                            let response = await originalImplementation.passwordResetPOST(input);
+                            
+                            // Then we check if it was successfully completed
+                            if (response.status === "OK") {
+                                // TODO: post password reset logic
```

Comment:
What else is there in the resposne? Can you get the userId for example?

Reply:
The response just contains `status` which can either be `OK` or the error when the reset password token is invalid